api/server.go

```diff
@@ -6,53 +6,63 @@ import (
 	"net/http"
 	"os"
 
+	"github.com/gin-gonic/gin"
+
 	"github.com/UTDNebula/nebula-api/api/configs"
 	"github.com/UTDNebula/nebula-api/api/routes"
-	"github.com/gin-gonic/gin"
-	"github.com/joho/godotenv"
 )
 
 func main() {
-	loadEnv()
-	loadMongo()
-	serveTraffic()
-}
-
-func loadEnv() {
-	err := godotenv.Load(".env")
-	if err != nil {
-		log.Print("There was no .env file found!")
-	}
-}
+	router := gin.Default()
 
-func loadMongo() {
-	configs.ConnectDB()
-}
+	// enable cors
+	/*router.Use(cors.New(cors.Config{
+	    AllowMethods:     []string{"GET", "POST", "OPTIONS", "PUT"},
+	    AllowHeaders:     []string{"*"},
+	    ExposeHeaders:    []string{"Content-Length"},
+	    AllowCredentials: false,
+	    AllowAllOrigins:  true,
+	    AllowOriginFunc: func(origin string) bool { return true },
+	    MaxAge:          86400,
+	}))*/
 
-func serveTraffic() {
-	router := gin.Default()
 	router.Use(CORS())
 
+	// connect to database
+	configs.ConnectDB()
+
+	// routes
 	routes.CourseRoute(router)
 	routes.DegreeRoute(router)
 	routes.ExamRoute(router)
 	routes.SectionRoute(router)
 	routes.ProfessorRoute(router)
+	routes.GradesRoute(router)
+
+	//router.OPTIONS("*", CORSOptionsHandler())
+
+	// @DEBUG
+	// router.GET("/", func(c *gin.Context) {
+	//     c.String(http.StatusOK, "Hello World!")
+	// })
 
-	port, exist := os.LookupEnv("PORT")
-	if !exist {
+	port := os.Getenv("PORT")
```

Comment:
I think these lines are already executed by LookupEnv. If the environment variable doesn't exist, just set the port.